build.gradle

```
@@ -25,29 +25,43 @@ group 'bio.terra'
 version '0.0.16-SNAPSHOT'
 sourceCompatibility = JavaVersion.VERSION_1_8
 
+def OPENCENSUS_VERSION = '0.28.2'
+
 dependencies {
     implementation group: 'bio.terra', name: 'stairway', version: '0.0.37-SNAPSHOT'
+    implementation group: 'ch.qos.logback.contrib', name: 'logback-json-classic', version: '0.1.5'
+    implementation group: 'ch.qos.logback.contrib', name: 'logback-jackson', version: '0.1.5'
+    implementation group: 'com.google.cloud', name: 'google-cloud-core', version: '1.94.0'
+    implementation group: 'com.google.apis', name: 'google-api-services-logging', version: 'v2-rev662-1.18.0-rc'
     implementation group: 'io.kubernetes', name: 'client-java', version: '10.0.0'
-    implementation group: "io.opencensus", name: "opencensus-api", version: "0.28.2"
+    implementation group: 'io.opencensus', name: 'opencensus-api', version: OPENCENSUS_VERSION
     implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.11'
```

version controlled by spring:

```suggestion
    implementation group: 'org.apache.commons', name: 'commons-lang3'
```